### DISCUSSION TITLE
Email editing

### DISCUSSION THREAD

**Discussion by opiepl** [pterodactyl/panel#2999] [Support] [answered]
Hi, I want to edit this part of email, where can I find it?
![mail](https://user-images.githubusercontent.com/75224417/104385408-77691300-5533-11eb-8342-851446760719.png)

**Comment by trenutoo** [comment, marked as answer]
You can edit the [email.blade.php](https://github.com/pterodactyl/panel/blob/6893f9bfcdc858a17f2e7144967ba84917a7ff9f/resources/views/vendor/notifications/email.blade.php) file to change the contents of the email
